### ISSUE TITLE
finetuning on test set?

### ISSUE THREAD

**Issue by cnwangy** [zjc062/mind-vis#5]
In stageA2_mbm_finetune.py, I found finetuning seemed to happen on test set. Am I mistaken?
https://github.com/zjc062/mind-vis/blob/7aece5706ea77b0ff254da70c14d448f81493e77/code/stageA2_mbm_finetune.py#L122

https://github.com/zjc062/mind-vis/blob/7aece5706ea77b0ff254da70c14d448f81493e77/code/stageA2_mbm_finetune.py#L137

https://github.com/zjc062/mind-vis/blob/7aece5706ea77b0ff254da70c14d448f81493e77/code/stageA2_mbm_finetune.py#L157

**Comment by zjc062** [comment]
Hi thank you for your question! 

Yes, we used **only the fMRI data** from the test set to finetune the encoder-decoder structure. It's a common practice in "pre-train then finetune" paradigm. There will be no data leakage as long as the paired data is not seen by the model. This method was also used in previous literature [(Beily 2019)](https://arxiv.org/abs/1907.02431).

Feel free to let us know if there are any other questions!

**Comment by cnwangy** [comment]
OK. Many thanks!
